core/README.md

```diff
@@ -54,8 +42,25 @@ lerna bootstrap
 The `lerna run` command runs the given script in every package referenced in the `lerna.json` file.
 ```bash
 lerna run bundle
+// or non-minified and watcher:
+lerna run bundle-develop
 ```
 
+**Link manually and build**
+
+If you do not want to use lerna.
+To manually link local modules, follow these steps:
+
+1. Go to `cd {luigi-core-root-folder}`.
+2. Go to `cd public` and run `npm link` once. It links `@kyma-project/luigi-core` to the global `node_modules`.
+3. Run `npm run bundle -- --watch`. It builds and updates on change.
+4. Navigate to `cd {app-folder}/node_modules`.
+5. Remove or move the `@kyma-project/luigi-core` and `@kyma-project/luigi-client` folders.
+6. Run `npm link luigi-core`. This links `@kyma-project/luigi-core` to the global `node_modules`.
+7. The autoreload of your application only updates the application. Select the CMD + R key combination to reflect the changes of the linked module on the website.
```

Comment:
The auto reload of your application only updates the application.

Reply:
Removed this block, as we decided yesterday just require to use lerna instead of confusion by to many options.